portality/static/widget/fq_widget_depends_compiled.js

```diff
@@ -1026,12 +1026,12 @@ function dateHistogramShowLess(options, context, facet) {
 }
 
 var licenceMap = {
-    "CC BY" : ["https://doaj.org/static/doaj/images/cc/by.png", "http://creativecommons.org/licenses/by/3.0/"],
-    "CC BY-NC" : ["https://doaj.org/static/doaj/images/cc/by-nc.png", "http://creativecommons.org/licenses/by-nc/3.0/"],
-    "CC BY-NC-ND" : ["https://doaj.org/static/doaj/images/cc/by-nc-nd.png", "http://creativecommons.org/licenses/by-nc-nd/3.0/"],
-    "CC BY-NC-SA" : ["https://doaj.org/static/doaj/images/cc/by-nc-sa.png", "http://creativecommons.org/licenses/by-nc-sa/3.0/"],
-    "CC BY-ND" : ["https://doaj.org/static/doaj/images/cc/by-nd.png", "http://creativecommons.org/licenses/by-nd/3.0/"],
-    "CC BY-SA" : ["https://doaj.org/static/doaj/images/cc/by-sa.png", "http://creativecommons.org/licenses/by-sa/3.0/"]
```

Comment:
Hmm, fair enough. Ok, we can compare if it comes to upgrading it. We need to get it out so people can use it, and if it does prove popular enough to warrant a redesign with the next redesign of the site we will have to compare the contents of this file, or simply rewrite it using the same process we used this time. Hopefully made a lot easier by just looking at changes to this file with git.